classes/class-pmpro-subscription.php

```diff
@@ -146,9 +216,13 @@ public function __construct( $subscription = null ) {
 
 		if ( ! empty( $subscription_data ) ) {
 			foreach ( $subscription_data as $arg => $value ) {
-				// Enforce integers for properties.
+				// Enforce integers/floats for properties.
```

Comment:
@dparker1005 I just added this for you since we're introducing floats

Reply:
Awesome, great catch. Thanks!